getgather/mcp/patterns/astro-otp.html

```diff
@@ -0,0 +1,8 @@
+<html gg-domain="astronauts.id">
+  <head>
+    <title>Astro OTP</title>
+  </head>
+  <body>
+    <input type="text" placeholder="One-time code" gg-match='[data-testid="otpInput0"]' />
```

Comment:
This likely needs to optionally match an error message (e.g. when the code is incorrect, or time out).